recipes/dcase2023_task4_baseline/local/sed_trainer.py

```diff
@@ -172,6 +176,16 @@ def on_train_start(self) -> None:
                                                                 "training_codecarbon"))
         self.tracker_train.start()
 
+        # Remove for debugging. Those warnings can be ignored during training otherwise.
+        # to_ignore = []
```

Comment:
I guess. The messages set to be ignored afterwards are polluting unnecessarily the printed logs (disrupting the progress bar for instance) because the first validation epochs have close to 0 psds scores, which triggers warnings from psds_eval.

But in this comment we let the possibility to stop ignoring those warnings if needed.